### ISSUE TITLE
Will it receive update to v10?

### ISSUE THREAD

**Issue by CapivaraTupiniquim** [cdverrett94/FVTT-overhead-tile-fade-for-all-tokens#11]
Hello! I love this module and used it in v9, but now with v10 it stopped working and I would really like to be able to use the overhead tiles as before (the new version of Foundry still doesn't have the options that this module has). If you need anything, let us know!

**Comment by CapivaraTupiniquim** [comment]
![image](https://user-images.githubusercontent.com/67530921/234161128-81ca2665-8a05-4fbf-b52b-d0b9baf8c21f.png)

**Comment by GambetTV** [comment]
I'd like to second this. This was a pretty vital module for me, and literally the only way I've been able to continue on without it is to just stop using roof tiles.

**Comment by CapivaraTupiniquim** [comment]
> I'd like to second this. This was a pretty vital module for me, and literally the only way I've been able to continue on without it is to just stop using roof tiles.

Yes! I use it to control the treetops, so when there's a token down, the top disappears and only the trunk and the shadow of the leaves remain. If you're interested, a member on the Foundry League of Extraordinary Discord charged $50 to update.
